tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,34 @@
+## About the Task
+
+`Feature-Extraction` starts from initial set of measured data and builds derived values (features) intended to be informative and non-redundant, facilitating the subsequent learning and generalization steps in text, image & audio processing.
+The Hub contains over 2000 models for [feature-extraction](https://huggingface.co/models?pipeline_tag=feature-extraction&p=2&sort=downloads).
```

Comment:
```suggestion
```
I think we'd have to repeatedly update this so it's not needed, people can filter for themselves 🙂